src/test/lib/ApiTest.cpp

```diff
@@ -4227,6 +4227,108 @@ void QuicTest_QUIC_PARAM_CONN_STATISTICS_V2_PLAT(MsQuicRegistration& Registratio
     }
 }
 
+
+void QuicTest_QUIC_PARAM_CONN_ORIG_DEST_CID(MsQuicRegistration& Registration, MsQuicConfiguration& ClientConfiguration) {
+    //
+    // This is the unit test for checking to see if a server has the correct original dest CID.
+    //
+    TestScopeLogger LogScope0("QUIC_PARAM_CONN_ORIG_DEST_CID");
+    {
+        MsQuicConnection Connection(Registration);
+        TEST_QUIC_SUCCEEDED(Connection.GetInitStatus());
+        TEST_QUIC_SUCCEEDED(
+        MsQuic->ConnectionStart(
+            Connection.Handle,
+            ClientConfiguration,
+            QUIC_ADDRESS_FAMILY_INET,
+            "localhost",
+            4433));
+        //
+        // 8 bytes is the expected minimum size of the CID.
+        //
+        uint32_t SizeOfBuffer = 8;
+        uint8_t Buffer[8]; 
+        TestScopeLogger LogScope1("GetParam test success case");
+        TEST_QUIC_STATUS(
+            QUIC_STATUS_SUCCESS, 
+            Connection.GetParam(
+                QUIC_PARAM_CONN_ORIG_DEST_CID,
+                &SizeOfBuffer, 
+                Buffer
```

Comment:
What is expected value of this `Buffer`?
Use `memcmp` to check if `Buffer` is filed by that.
Or if it is filled by random value within core code, at least compare if the output is not zero.
```c
uint8_t Buffer[8] = {0};
uint8_t ZeroBuffer[8] = {0};
GetParam
TEST_TRUE(memcmp(Buffer, ZeroBuffer, sizeof(Buffer)) != 0);
```

Reply:
I chatted with Nick about not going too crazy with the tests. And yes, it will be filled with random values in the core code. I can add non-null check, but there is a 1 / 2^64 chance the CID will actually be all 0s LOL